### ISSUE TITLE
Can not connect to postgresql, without connection errors

### ISSUE THREAD

**Issue by marekkrivan** [typeorm/typeorm#6243]
**Issue type:**

[ ] question
[x] bug report
[ ] feature request
[ ] documentation issue

**Database system/driver:**

[ ] `cordova`
[ ] `mongodb`
[ ] `mssql`
[ ] `mysql` / `mariadb`
[ ] `oracle`
[x] `postgres`
[ ] `cockroachdb`
[ ] `sqlite`
[ ] `sqljs`
[ ] `react-native`
[ ] `expo`

**TypeORM version:**

[x] `latest`
[ ] `@next`
[ ] `0.x.x` (or put your version here)

**Steps to reproduce or a small repository showing the problem:**

Hi everybody,

I created Typeform project, but I can not connect to Postgres. 

I followed step by step from the documentation.

 1. npm install typeorm -g
 2. typeorm init --name MyProject --database postgres
 3. cd MyProjectnpm install

then I runed command npm start and this is output
> MyProject@0.0.1 start /Users/marek/Documents/works/test/MyProject
> ts-node src/index.ts

marek@Mareks-MacBook-Pro MyProject % 

The output is without errors but the script can not connect, I know it because the expected output is "console.log("Connected.");"

This is code

    createConnection().then(async connection => {
      console.log("Connected.");
    }).catch(error => console.log("error: " + error));

This part "console.log("error: " + error)" returned nothing.

I change the other Postgres server and the output was the same.
I change parameters type in ormconfig.json to MySQL everything was OK.

Has somebody an Idea? 

Thank you.

**Comment by sh0cked** [comment]
https://github.com/typeorm/typeorm/issues/5949#issuecomment-619531559

**Comment by marcushultgren** [comment]
> [#5949 (comment)](https://github.com/typeorm/typeorm/issues/5949#issuecomment-619531559)
> Update pg to > pg@8.0.3

That solved it, thanks! Strange that it does not print any error if you keep it at "pg": "^7.3.0".

**Comment by imnotjames** [comment]
Closing this as a duplicate of #5949 - cheers!

**Comment by CelestialMusa** [comment]
What exactly was the problem? I'm on pg 8.0.3 but still see no errors on when trying to connect to the database.

**Comment by ubershmekel** [comment]
@CelestialMusa I personally did not get an error. My original code was able to connect to an older version of postgres (PostgreSQL 11 on GCP Cloud SQL). When connecting to the new Postgres 13, typeorm would just hang on `createConnection` and never connect. I added a timeout to detect that scenario. I saw `psql` connecting fine, so started fiddling with ports. Then I luckily found this bug and solution. Updated `pg` and it all worked like a charm.
